pkg/gds/certman/iface.go

```diff
@@ -0,0 +1,12 @@
+package certman
+
+import "sync"
+
+// Service defines the CertMan go routine interface for outside users to interact with
+// the certificate manager directly.
+type Service interface {
+	Run(*sync.WaitGroup) error
+	Stop()
+	CertManager()
+	HandleCertificateRequests()
```

Comment:
Is there a reason we're excluding the `HandleCertificateReissuance` from the interface?

Reply:
Good catch, I can add that back in.